powershell/admin-center/Microsoft.PowerApps.Administration.PowerShell.Samples.psm1

```diff
@@ -1610,6 +1610,61 @@ function ExcludeInsightsForManagedEnvironmentsInWeeklyEmailDigest
     Write-Host "Excluded insights for the specified environment in weekly email digest."    
 }
 
+function SetSolutionCheckerEnforcement
```

Comment:
nit: These samples aren't specific to Manage Environments so I would use a more specific name, e.g., SetManagedEnvironmentSolutionCheckerEnforcementSettings

Reply:
I suffixed it with "Settings" because we might add more, e.g., the warning/blocked email thing.